panic: Not implemented

I tried your library, maybe I did something wrong, please give me an advice: I get "panic: Not implemented". This is my code

```
	sort.Strings(cache)

	b:= dawg.New()

	for _, c := range cache {
		b.Add(c)
	}

	return b.Finish()
```

cache is a string slice with 10M strings (domain names). 

Stacktrace:
```
panic: Not implemented: 10000001

goroutine 3 [running]:
log.Panicf(0x9d2ba7, 0x13, 0xc000159930, 0x1, 0x1)
        /usr/lib/go/src/log/log.go:358 +0xdd
github.com/smhanov/dawg.unsignedLength(0x989681, 0x0)
        /home/x/go/pkg/mod/github.com/smhanov/dawg@v0.0.0-20201027134228-64393ae07b2f/disk.go:511 +0x130
github.com/smhanov/dawg.(*dawg).Write(0xc00014c000, 0xa11fe0, 0xc1151ea000, 0x0, 0x0, 0x0)
        /home/x/go/pkg/mod/github.com/smhanov/dawg@v0.0.0-20201027134228-64393ae07b2f/disk.go:113 +0x514
github.com/smhanov/dawg.(*dawg).Finish(0xc00014c000, 0x0, 0x0)
        /home/x/go/pkg/mod/github.com/smhanov/dawg@v0.0.0-20201027134228-64393ae07b2f/dawg.go:228 +0x250
```

EDIT: Tested with 200K list -> same result

--- a/disk.go
+++ b/disk.go
@@ -482,6 +482,11 @@ func writeUnsigned(w *bitWriter, n uint64) {
 		w.WriteBits((n>>14)&0x7f|0x80, 8)
 		w.WriteBits((n>>7)&0x7f|0x80, 8)
 		w.WriteBits(n&0x7f, 8)
+	} else if n < 0xfffffff {
+		w.WriteBits((n>>21)&0x7f|0x80, 8)
+		w.WriteBits((n>>14)&0x7f|0x80, 8)
+		w.WriteBits((n>>7)&0x7f|0x80, 8)
+		w.WriteBits(n&0x7f, 8)
 	} else {
 		// could go further
 		log.Panic("Not implemented")
@@ -507,6 +512,8 @@ func unsignedLength(n uint64) uint64 {
 		return 2
 	} else if n < 0x1fffff {
 		return 3
+	} else if n < 0xfffffff {
+		return 4
 	}
 	log.Panicf("Not implemented: %d", n)
 	return 0

